lib/commands/check.js

```diff
@@ -16,7 +16,16 @@ var onCommand = function (args) {
     args.output = getFormatter(args.output);
   }
 
-  var pkgPath = Path.join(process.cwd(), 'package.json');
+  if (!args.packagePath){
+    var pkgPath = Path.join(process.cwd(), 'package.json');
+  }
+  else if (!args.packagePath.includes('package.json')) {
```

Comment:
FWIW, I think `Array#includes()` was added in Node 6.5, so this may fail in anything lower: http://node.green/#ES2015-built-ins-well-known-symbols-Symbol-match--String-prototype-includes